src/tests/rockets/Rockets.test.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { BrowserRouter } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import store from '../../redux/store';
+import Rockets from '../../components/rockets/Rockets';
+
+test('Check if dragon component is rendered correctly', () => {
+  const tree = render(
+    <Provider store={store}>
+      <BrowserRouter>
+        <Rockets />
+      </BrowserRouter>
+    </Provider>,
+  );
```

Comment:
- [ ] I think you are trying to test the rendering, I will encourage you to use snapshot, at the moment the 5 tests(these are the only tested because they use the correct extension, others are using the wrong extension) fails and I will encourage you to double check them, you inly need to import fuctions/component you want to test, you don't need the react-router-dom or provider here.

Let me know if it is unclear, the extension should be [name of the file].test.js

Reply:
@Olamarx Can you please elaborate on this changes? we are kind of lost here.

If possible will you be able to join a call with us for full context please?

Reply:
I am talking about the test you wrote, let’s join call if possible, please.

Reply:
I will send a link now

Reply:
Here:
https://meet.google.com/jht-derw-bzp

@Olamarx

Reply:
Kindly invite me on Zoom.

Reply:
I've sent you an invite on zoom